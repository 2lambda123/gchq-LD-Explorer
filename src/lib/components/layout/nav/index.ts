/* (c) Crown Copyright GCHQ */

export { default as SubNav } from './SubNav.svelte';
export { default as SubNavItem } from './SubNavItem.svelte';
export { default as SubNavLayout } from './SubNavLayout.svelte';
export { default as SideNav } from './SideNav.svelte';
