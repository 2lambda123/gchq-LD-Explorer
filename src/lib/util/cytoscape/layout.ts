/* (c) Crown Copyright GCHQ */

export default { name: 'cose', padding: 200, animate: false, componentSpacing: 10 };
