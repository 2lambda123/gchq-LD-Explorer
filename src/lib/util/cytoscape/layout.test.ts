/* (c) Crown Copyright GCHQ */

import layout from './layout';

describe('cytoscape', () => {
	describe('layout', () => {
		it('returns the layout config', () => {
			expect(layout).toMatchSnapshot();
		});
	});
});
