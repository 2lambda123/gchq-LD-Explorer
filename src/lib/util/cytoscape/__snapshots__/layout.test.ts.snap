// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`cytoscape > layout > returns the layout config 1`] = `
{
  "animate": false,
  "componentSpacing": 10,
  "name": "cose",
  "padding": 200,
}
`;
